MOCC/src/configurations/NewPyTango/HealthMonitorDS.py

```diff
@@ -0,0 +1,39 @@
+from flask import Flask, jsonify
+from flasgger import Swagger
+
+
+app = Flask(__name__)
+app.config['SWAGGER'] = {
+    'title': 'Health Monitor API',
+    'uiversion': 2
+}
+Swagger(app)
```

Comment:
Take a look at http://flasgger.pythonanywhere.com/ (any of the live demos)
All REST functions provided by all the services we write should have a Swagger definition.
This includes the device servers, like the health monitor, as well as the config manager.